classes/Http/Controller/TalkController.php

```diff
@@ -191,6 +193,10 @@ public function editAction(Request $req)
             'slides' => $talk_info['slides'],
             'other' => $talk_info['other'],
             'sponsor' => $talk_info['sponsor'],
+            'tags' => array_reduce($talk_info['tags'], function ($response, $item) {
```

Comment:
Can we add a type-hint for `$item`? Looks like it is either an `array` or an object implementing `ArrayAccess`, doesn't it?